### DISCUSSION TITLE
How to play alarm sound on android device also when device is on mute (DND) like critical alert in iOS

### DISCUSSION THREAD

**Discussion by galdan** [invertase/react-native-firebase#8164] [Q&A]
Hi ,In iOS there is option for send **critical** push.
```
 "apns": {
    "headers": {
      "apns-priority": "10",
      "apns-push-type": "alert"
    },
    "payload": {
      "aps": {
        "alert": {
          "title": "xxxx",
          "body": "yyyyy"
        },
        "sound": {
          "critical": 1,
          "name": “alarm.mp3”,
          "volume": 1.0
        },
}
```
in the sound dictionary you can pass critical , sound name and volume.
How can I make the same logic on android devices? I didn't found the official Documentation about it.
I am looking for the server json struct . and also what to do in the app side will be great (less important for now but can also be great ).
Thanks for your help!

**Comment by mikehardy** [comment]
Hi there, this is a generic question about FCM / android - I'd suggest posting on stackoverflow with appropriate tags

**Reply by galdan** [reply]
@mikehardy Thanks!
I did it, also if someone is know how to do it  from this community - its will be great!
